docs/helm_testing.md

```diff
@@ -36,6 +36,9 @@ The API Test Job includes the following test steps:
 - Wait for the API to be ready.
 - Execute the tests using the `subscribe-submit-test` script.
 
+Note, the API test uses the `"chronicle-system"` namespace. For more details, see
+[here](./namespaces.md#chronicle-system).
```

Comment:
```suggestion
Note, the API test uses the
[`chronicle-system` namespace](./namespaces.md#chronicle-system).
```
Some style guides advise against "here" links so I try to avoid them at least when it's easy to.